plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/explorer/OpenShiftExplorerLabelProvider.java

```diff
@@ -80,7 +84,9 @@ public String getText(Object element) {
 
 	@Override
 	public Image getImage(Object element) {
-		if (element instanceof Deployment) {
+		if (element instanceof OpenShiftProjectUIModel) {
```

Comment:
This should be the ProjectAdapter interface this model implements instead of the concrete class.